plugins/reporters/asciidoc/src/funTest/kotlin/HtmlTemplateReporterFunTest.kt

```diff
@@ -38,9 +43,24 @@ class HtmlTemplateReporterFunTest : StringSpec({
 
         reportContent.patchAsciiDocTemplateResult() shouldBe patchExpectedResult(
             expectedResult,
-            custom = mapOf("<REPLACE_ASCIIDOCTOR_VERSION>" to reporter.asciidoctor.asciidoctorVersion())
+            custom = mapOf(
+                "<REPLACE_ASCIIDOCTOR_VERSION>" to AsciiDocTemplateReporter.asciidoctor.asciidoctorVersion()
+            )
         )
     }
+
+    "Reports can be generated in parallel" {
+        val reportCount = 16
+        val input = ReporterInput(ORT_RESULT)
+
+        withContext(Dispatchers.IO) {
+            (1..reportCount).map {
+                async {
+                    HtmlTemplateReporterFactory.create().generateReport(input, tempdir())
+                }
+            }.awaitAll()
+        }
```

Comment:
Hm, as this test cannot reproduce the issue, I'd slightly tend to drop it (but up to you).
Otherwise LGTM.

Reply:
I tend to agree. If there is no confidence added by this test, it's just useless code that we need to carry around, and I'd also drop it.

Reply:
The test has been dropped.

Reply:
Maybe in the future something like https://github.com/cmu-pasta/fray could help to reproduce issues in such cases.